devicetypes/smartthings/aeon-siren.src/README.md

```diff
@@ -0,0 +1,37 @@
+# Aeon Siren
+
+Cloud Execution
```

Comment:
That refers to the location of the code execution which in this case is **cloud** and not **local**.